File: Lab 2/Prelab2.c
#include <stdio.h>

int main(void)
{
    int valueX;
    int valueY;

    printf("%s","Enter 2 integers separated by spaces:");
    scanf("%d%d", &valueX, &valueY);

    printf("*** Initial values ***\n");
    printf("X = %d\n", valueX);
    printf("Y = %d\n\n", valueY);
    
    printf("*** Calculated values ***\n");
    printf("X times Y = %d\n", (valueX * valueY));
    printf("X divided by Y = %d\n", (valueX/valueY));
    printf("X plus 1, quantity times Y = %d\n\n", ((valueX+1)*valueY));

    printf("*** Final values ***\n");
    printf("X = %d\n", valueX);
    printf("Y = %d\n", valueY);

}

File: Lab 2/Prelab2.py
# Brett M.
# Sept. 1, 2021
# CS 1050 - 01 Lab 2 except in python
# Take input from the user and perform basic math with the integers

user_numbers = input("Enter 2 integers separated by a space: ")
numbers_split = user_numbers.split()
x = int(numbers_split[0])
y = int(numbers_split[1])

print("*** Initial values ***")
print("X = {}".format(x))
print("Y = {}\n".format(y))

print("*** Calculated values ***")
print("X times Y = {}".format(x*y))
print("X divided by Y = {}".format(int(x/y))) # int() only used to copy sample output
print("X plus 1, quantity times Y = {}\n".format((x+1)*y))

print("*** Final values ***")
print("X = {}".format(x))
print("Y = {}".format(y))
